modules/core/nuxeo-core-management/src/main/resources/OSGI-INF/runtime-management-contrib.xml

```diff
@@ -5,8 +5,8 @@
     point="factories">
     <factory name="managementStatusesFactory"
       class="org.nuxeo.ecm.core.management.StatusesManagementFactory" />
-    <factory name="EventMonitoring"
-      class="org.nuxeo.ecm.core.management.events.EventMonitoringFactory" />
+    <!-- factory name="EventMonitoring"
+      class="org.nuxeo.ecm.core.management.events.EventMonitoringFactory" / -->
```

Comment:
Do we want to keep this commented code?

Reply:
Should go in the JIRA upgrade notes/documentation to activate it back (?)

Reply:
removed and comment added to the ticket